### PR TITLE
New Gaggia Logo and new Heizlogo / Heizlogo project Logo based

### PR DESCRIPTION
**Pull request by ibaxx**
userconfig extended with new Logos
#define HEATINGLOGO 3              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia, 3 = new Gaggia with new Heizlogo, 4 = Rancilio with new Heizlogo, 5 = Old Gaggia with new Heizlogo

 the new Heizlogo is based on the clever coffee Logo

updated userconfig_sample.ini and updated display.h and icon.h

### CODE REVIEW

**Comment by kjyv** [comment]
Could you add a photo of the new logos, what is the reasoning to add these?

**Comment by ibaxx** [comment]
War den betreffenden Tag im Chat gewesen.

image

**Comment by ibaxx** [comment]
Leider etwas staubig 😉

**Comment by kjyv** [comment]
Ok thanks. I guess we can easily include it once we only support esp32 (as it has more memory)

**Comment by murmeltier08** [comment]
Can me merge it now?

**Comment by ibaxx** [comment]
You are welcome

**Comment by SphaeroX** [comment]
Didnt tested it but the Sample config should be #define HEATINGLOGO 0

Not #define HEATINGLOGO 3

File: src/userConfig_sample.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ enum MACHINE {
 #define DISPLAYTEMPLATE 3          // 1 = Standard Display Template, 2 = Minimal Template, 3 = only Temperatur, 4 = Scale Template, 20 = vertical Display see git Handbook for further information
 #define DISPLAYROTATE U8G2_R0      // rotate display clockwise: U8G2_R0 = no rotation; U8G2_R1 = 90°; U8G2_R2 = 180°; U8G2_R3 = 270°
 #define SHOTTIMER 1                // 0 = deactivated, 1 = activated 2 = with scale
-#define HEATINGLOGO 0              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia
+#define HEATINGLOGO 3              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia, 3 = new Gaggia with new Heizlogo, 4 = Rancilio with new Heizlogo, 5 = Old Gaggia with new Heizlogo
```
Comment (LoQue90) [inline src/userConfig_sample.h:37]: #define HEATINGLOGO 0              // 0 = deactivated, 1 = Rancilio, 2 = Gaggia, 3 = new Gaggia with new heating logo, 4 = Rancilio with new heating logo, 5 = Old Gaggia with new heizlogo

gaggia and  new gaggia are two diffenrent mashines? if so, do they have clear names we can add?
define some with "new" is IMO a realy bad way. as you said you tryed to match the heating logo with the clevercoffee logo, so maybe we find way clear way to describe them. "generic heat logo" and clevercoffee heat logo".
Im open for any ideas here

File: src/display.h
Comment (LoQue90) [inline src/display.h:1]: please go with complete english comments and varibales
Heiz = heat

File: src/icon.h [hunk 2]
```diff
@@ -368,6 +368,29 @@ static const unsigned char Heiz_Logo[] = {
     0xff, 0x1f, 0xff, 0xff, 0xff, 0xff, 0x1f, 0xff, 0xff, 0xff, 0xff, 0x1f,
     0x00, 0x00, 0x00, 0x00, 0x00
 };
+static const unsigned char Heiz_Logo_new[] = {
```
Comment (LoQue90) [inline src/icon.h:371]: e.g.:
static const unsigned char heat_Logo_clevercoffee[] = {

File: src/icon.h [hunk 3]
```diff
@@ -472,3 +495,30 @@ static const unsigned char  Gaggia_Classic_Logo[] = {
     0x00, 0x60, 0x00, 0x00, 0x00, 0x20, 0x00, 0x20, 0x00, 0x00, 0x00, 0x20,
     0x00, 0xe0, 0xff, 0xff, 0xff, 0x3f
 };
+static const unsigned char  Gaggia_Classic_Logo_new[] = {
```
Comment (LoQue90) [inline src/icon.h:498]: also now "new" in vars

**Review by LoQue90 — CHANGES_REQUESTED** [review]

**Comment by LoQue90** [comment]
@ibaxx could you rebase this pr to our current master?
and this should be in the end only 1 comment pls :)

File: src/icon.h (hunk 3 above)
Comment (ibaxx) [reply src/icon.h:498]: Hi, i thought the Logos where already merged to the Master?
I have used Heiz, because there was Heiz used in the old Code.

Comment (LoQue90) [reply src/icon.h:498]: Hey, no this is not merged, the pr would be closed then.

Yes, "Heiz" was already used, still not nice.
So I would change that in your pr and later on in the whole code.
But feel free to change that globaly.

**Pull request closed without merging**